docker-compose.yml

```diff
@@ -20,54 +19,37 @@ networks:
 
 services:
   # OpenSearch store - node1
-  opensearch-node1: # This is also the hostname of the container within the Docker network (i.e. https://opensearch-node1/)
-    image: opensearchproject/opensearch:${VERSION} # Specifying the latest available image - modify if you want a specific version
-    container_name: opensearch-node1
-    environment:
-      - cluster.name=opensearch-cluster # Name the cluster
-      - node.name=opensearch-node1 # Name the node that will run in this container
-      - discovery.seed_hosts=opensearch-node1,opensearch-node2 # Nodes to look for when discovering the cluster
-      - cluster.initial_cluster_manager_nodes=opensearch-node1,opensearch-node2 # Nodes eligible to serve as cluster manager
-      - bootstrap.memory_lock=true # Disable JVM heap memory swapping
-      - "OPENSEARCH_JAVA_OPTS=-Xms512m -Xmx512m" # Set min and max JVM heap sizes to at least 50% of system RAM
-    ulimits:
-      memlock:
-        soft: -1 # Set memlock to unlimited (no soft or hard limit)
-        hard: -1
-      nofile:
-        soft: 65536 # Maximum number of open files for the opensearch user - set to at least 65536
-        hard: 65536
-    volumes:
-      - opensearch-data1:/usr/share/opensearch/data # Creates volume called opensearch-data1 and mounts it to the container
-    healthcheck:
-      test: ["CMD", "curl", "-f", "https://opensearch-node1:9200/_cluster/health?wait_for_status=yellow", "-ku ${USER}:${PASSWORD}"]
-      interval: 5s
-      timeout: 25s
-      retries: 4
-    ports:
-      - "9200:9200"
-      - "9600:9600"
-
-  # OpenSearch store - node2
-  opensearch-node2:
-    image: opensearchproject/opensearch:${VERSION} # This should be the same image used for opensearch-node1 to avoid issues
-    container_name: opensearch-node2
+  opensearch:
+    image: opensearchstaging/opensearch:3.0.0
```

Comment:
the current staging image `opensearchstaging/opensearch:3.0.0` only contains 1 plugin
```
opensearch-security
```

users might need other plugins for observability playground to work correctly

Reply:
I'll add documentation for this...